resources/js/pages/terms.tsx

```diff
@@ -0,0 +1,169 @@
+import { Head } from '@inertiajs/react';
+
+import { LegalNotice } from '@/common/components/LegalNotice';
+import { AppLayout } from '@/common/layouts/AppLayout';
+import type { AppPage } from '@/common/models';
+
+const Terms: AppPage = () => {
+  return (
+    <>
+      <Head title="Legal & Terms">
+        <meta
+          name="description"
+          content="Review RetroAchievements.org's terms of use, code of conduct, disclaimers, copyright policy, and privacy policy. Stay informed on how we handle content, links, and your personal data."
+        />
+      </Head>
+
+      <AppLayout.Main>
+        <h1 className="mb-4">Legal & Terms</h1>
+
+        <div className="flex flex-col gap-y-4">
+          <div className="rounded bg-embed p-4">
+            <div className="flex flex-col gap-y-6">
+              <div>
+                <h2 className="text-h4" id="conditions">
```

Comment:
I assume it's because of the sticky menu, but this link appears to go to the next paragraph.
![image](https://github.com/user-attachments/assets/9288324a-0bf5-4050-b6a1-99f4010b26bf)

Also verified this behavior is present outside of this PR.

Reply:
Tricky - might as well go ahead and fix this while we're here.

In latest, I've moved the `id` to an invisible div that has some offset so the user lands here:
![Screenshot 2024-08-28 at 6 54 21 PM](https://github.com/user-attachments/assets/e7d05784-79a4-475f-9172-86f82bf3c37e)